lib/Parser/OglParser.php

```diff
@@ -99,7 +99,7 @@ public function parseStream($handle, \Mesamatrix\Git\Commit $commit = null) {
                         $matrix->addGlVersion($glVersion);
                     }
 
-                    $line = $this->parseSection($glVersion, $matrix, $commit, $handle, $allSupportedDrivers);
+                    $line = $this->parseSection($glVersion, $matrix, $commit, $handle);
```

Comment:
I'll need to check on this one because it doesn't seem normal that the previous `if` block (l91) has the parameter, but not this one.